cmd/terramate/cli/cloud.go

```diff
@@ -239,12 +240,26 @@ func (c *cli) detectCloudMetadata() {
 		Str("head_commit", c.prj.headCommit()).
 		Logger()
 
-	if c.prj.prettyRepo() == "local" {
-		logger.Debug().Msg("skipping review_request and metadata for local repository")
+	prettyRepo := c.prj.prettyRepo()
+	headCommit := c.prj.headCommit()
+
+	c.cloud.run.metadata = &cloud.DeploymentMetadata{GitHeadCommitSHA: headCommit}
+	md := c.cloud.run.metadata
+
+	if commit, err := c.prj.git.wrapper.ShowCommitMetadata("HEAD"); err == nil {
+		setDefaultGitMetadata(md, commit)
+	} else {
+		logger.Warn().
+			Err(err).
+			Msg("failed to retrieve commit information from GitHub API")
+	}
+
+	if prettyRepo == "local" {
```

Comment:
I think this check must be moved to the beginning of the function, where it was before, because AFAIK we are not interested in metadata for local commits as we cannot show them or make any sense out of them in the TMC frontend, because they reference a local bare upstream directory.

Reply:
Ah okay, I see. I actually thought about where to put this check and my thought was that metadata like commit message might still be useful, but only if local data from local repositories is relevant at all.